.buildkite/pipeline.yml

```diff
@@ -0,0 +1,68 @@
+# steps:
```

Comment:
comments can be remove now? or the idea is to keep them for some reason? (minor)

Reply:
Sorry @mirkoAlic, they should be removed, I'll do that now :)